platform/src/main/java/org/stellar/anchor/platform/service/PaymentOperationToEventListener.java

```diff
@@ -97,6 +95,8 @@ public void onReceived(ObservedPayment payment) {
     }
 
     // Check if the payment contains the expected amount (or greater)
+    Instant paymentTime =
```

Comment:
💡 `FormatException` can be thrown. Since this is in a listener, we may do a try/catch block to protect against potential problems.

Reply:
Added in https://github.com/stellar/java-stellar-anchor-sdk/pull/475/commits/8d3bf604909523080260dd9e65ab5ff713179fde.